cloud-service-providers/aws/eks/nim-operator-setup.md

```diff
@@ -0,0 +1,142 @@
+# NVIDIA NIM Operator on AWS EKS:
+
+Please see the NIM Operator documentation before you proceed: https://docs.nvidia.com/nim-operator/latest/index.html
+This repository is dedicated to testing NVIDIA NIM Operator on AWS EKS (Elastic Kubernetes Service).
+
+## Cluster setup for inference:
+
+To install the pre-requisites for the NIM Operator, please follow the steps below:
+
+1: Install NVIDIA Device Plugin: Install NVIDIA device plugins to run GPU workloads. Check CUDA base image version for compatibility.
```

Comment:
Why are we installing the device plugin before the Operator? Is this necessary on EKS?